### ISSUE TITLE
请问大佬这是用CPU和内存在跑么？

### ISSUE THREAD

**Issue by liangzoey** [smthemex/ComfyUI_HunyuanAvatar_Sm#16]
为啥显卡占用没有CPU给我干到83°，内存100%啊？

**Comment by smthemex** [comment]
CPU卸载是这样的，我还没来及修改全量GPU，今天我修改一下代码

**Comment by Redbeard-himalaya** [comment]
@smthemex 辛苦啦 🙏🙏🙏
我刚才试了一下 "use_fp8 == true" + "cpu_offload == true"，发现 inference 刚开始的时候 GPU usage 为 0 并且持续了很长时间，过了这段之后 GPU usage 就变为 100%。GPU usage 正常以后 inference 速度约为 33s/it （width=512, height=512, image_size=512)

**Comment by Redbeard-himalaya** [comment]
@smthemex 辛苦啦 🙏🙏🙏
我刚才试了一下 "use_fp8 == true" + "cpu_offload == true"，发现 inference 刚开始的时候 GPU usage 为 0 并且持续了很长时间，过了这段之后 GPU usage 就变为 100%。GPU usage 正常以后 inference 速度约为 33s/it （width=512, height=512, image_size=512)

2025-06-20 10:22:49.940 | INFO | /ComfyUI/custom_nodes/ComfyUI_HunyuanAvatar_Sm.hymm_sp.sample_inference_audio:predict:190 -
prompt: Authentic, Realistic, Natural, High-quality, Lens-Fixed, A girl sits by a table in a room.
image_path:
audio_path: /ComfyUI/input/audio_449257_temp.wav
negative_prompt: Aerial view, aerial view, overexposed, low quality, deformation, a poor composition, bad hands, bad teeth, bad eyes, bad limbs, distortion, blurring, Lens changes
seed: 80538225
fps: 25.0
infer_steps: 50
target_height: 512
target_width: 512
target_length: 129
guidance_scale: 7.5

Expanding inputs for image tokens in LLaVa should be done in processing. Please add patch_size and vision_feature_select_strategy to the model's processing config or set directly with processor.patch_size = {{patch_size}} and processor.vision_feature_select_strategy = {{vision_feature_select_strategy}}. Using processors without these attributes in the config is deprecated and will throw an error in v4.47. Starting from v4.46, the logits` model output will have the same type as the model (except at train time, where it will always be FP32)
100% 50/50 [17:07<00:00, 20.55s/it]
z.shape: torch.Size([1, 16, 33, 64, 64]) 8 torch.float32
z.shape temporal_tiled_decode: torch.Size([1, 16, 33, 64, 64]) 8
2025-06-20 10:47:18.816 | INFO | /ComfyUI/custom_nodes/ComfyUI_HunyuanAvatar_Sm.hymm_sp.sample_inference_audio:predict:237 - Success, time: 1468.509444952011
Prompt executed in 1899.97 seconds

**Comment by smthemex** [comment]
CPU卸载开启之后,llm和clip 包括transformer都会移到CPU,这个要固态硬盘会好点,实际推理的时候,llm和clip会临时移到GPU推理,完成后卸载回CPU,这个方法OOM主要是一个批次是33帧,然后VAE那里最容易爆,大显存可以关闭tile,效果可能会好点
